catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/PaimonCatalogPropertiesMetadata.java

```diff
@@ -128,13 +128,13 @@ public class PaimonCatalogPropertiesMetadata extends BaseCatalogPropertiesMetada
                 "Gravitino Paimon catalog jdbc user",
                 false /* immutable */,
                 null /* defaultValue */,
-                true /* hidden */),
+                false /* hidden */),
```

Comment:
Is there any test to cover this change?

Reply:
no,  is it nessecery to add the specific test for Paimon JDBC catalog hidden properties?

Reply:
Is there a Paimon-JDBC catalog IT? If so, it would be easy to add the properties assert tests. Otherwise, we should supplement the IT.

Reply:
I think there is no need to add this  to IT, since it's minor changes

Reply:
i checked the MysqlCatalog, PGCatalog, and tec. None of them tested whether the jdbc user and password were hidden or not.

I think it is ok this pr can not include the test of this modification, but a new pr can be created to specifically improve them

Reply:
ok